composer.json

```diff
@@ -29,6 +29,7 @@
 	},
 	"minimum-stability": "dev",
 	"config": {
+		"autoloader-suffix": "10upAutoshareForTwitterV111",
```

Comment:
I'm fine with this approach but feels like we need to document this in our [release instructions](https://github.com/10up/autoshare-for-twitter/blob/develop/CONTRIBUTING.md#release-instructions), as I'm assuming we'll want to bump the version number here anytime we release a new version.

Reply:
Done. Thanks for pointing this out.